src/compiler/emitter.ts

```diff
@@ -3,6 +3,190 @@
 
 /* @internal */
 namespace ts {
+    export const defaultInitCompilerOptions: CompilerOptions = {
+        module: ModuleKind.CommonJS,
+        target: ScriptTarget.ES3,
+        noImplicitAny: true,
+        outDir: "built",
+        rootDir: ".",
+        sourceMap: false,

```

Comment:
Because it should be easy to switch otherwise people might know if it even exists. In VSCode they also show this as the default.